pages/agenda.tsx

```diff
@@ -26,39 +39,102 @@ export default function Agenda() {
+              <AgendaCard hour='16:40' isBreak={false} speakerName={s('noah.name')} talkTitle={s('noah.talk_title')} icon='/icons/microphone.svg' position='left' link={s('noah.id')}/>
+              <AgendaCard hour='17:20' isBreak={false} speakerName={s('francesco.name')} talkTitle={s('francesco.talk_title')} icon='/icons/microphone.svg' position='right' link={s('francesco.id')}/>
+              <AgendaCard hour='18:30' isBreak={true} breakTitle="Aperitivo" icon='/icons/drink.svg' position='left'/>
+            </>
+          )}
+
+          {trackSelected === 'Alpha2' && (
+            <>
+              <AgendaCard hour='8:30' isBreak={true} breakTitle='Open gate' icon='/icons/pencil.svg' position='right'/>
+              <AgendaCard hour='9:50' isBreak={true} breakTitle='Morning intro' icon='/icons/microphone.svg' position='left'/>
+              <AgendaCard hour='10:00' isBreak={false} speakerName={s('nathan.name')} talkTitle={s('nathan.talk_title')} icon='/icons/microphone.svg' position='right' link={s('nathan.id')}/>
+              <AgendaCard hour='10:45' isBreak={false} speakerName={s('tbd.name')} talkTitle={s('tbd.talk_title')} icon='/icons/microphone.svg' position='left' link={s('tbd.id')}/>
+              {/* <AgendaCard hour='10:40' isBreak={true} breakTitle='Coffee break' icon='/icons/coffee.svg' position='left'/> */}
+              <AgendaCard hour='11:30' isBreak={false} speakerName={s('tbd.name')} talkTitle={s('tbd.talk_title')} icon='/icons/microphone.svg' position='right' link={s('tbd.id')}/>
+              <AgendaCard hour='12:15' isBreak={true} breakTitle='Lunch break' icon='/icons/fork-and-knife.svg' position='left'/>
+              <AgendaCard hour='14:00' isBreak={true} breakTitle='Afternoon intro' icon='/icons/microphone.svg' position='right'/>
+              <AgendaCard hour='14:05' isBreak={false} speakerName={s('valeria.name')} talkTitle={s('valeria.talk_title')} icon='/icons/microphone.svg' position='left' link={s('valeria.id')}/>
+              <AgendaCard hour='14:45' isBreak={false} speakerName={s('santosh.name')} talkTitle={s('santosh.talk_title')} icon='/icons/microphone.svg' position='right' link={s('santosh.id')}/>
+              <AgendaCard hour='15:30' isBreak={false} speakerName={s('federico.name')} talkTitle={s('federico.talk_title')} icon='/icons/microphone.svg' position='left' link={s('federico.id')}/>
+              <AgendaCard hour='16:15' isBreak={false} speakerName={s('sabrina.name')} talkTitle={s('sabrina.talk_title')} icon='/icons/microphone.svg' position='right' link={s('sabrina.id')}/>
+              {/* <AgendaCard hour='16:00' isBreak={true} breakTitle='Coffee break & Giveaway' icon='/icons/coffee.svg' position='right'/> */}
+              <AgendaCard hour='17:00' isBreak={false} speakerName={s('andrei.name')} talkTitle={s('andrei.talk_title')} icon='/icons/microphone.svg' position='left' link={s('andrei.id')}/>
```

Comment:
```suggestion
              <AgendaCard hour='17:00' isBreak={false} speakerName={s('andrey.name')} talkTitle={s('andrey.talk_title')} icon='/icons/microphone.svg' position='left' link={s('andrey.id')}/>
```

Reply:
I did remove the not confirmed yet speakers. This commit is missing the speakers translation part but no error it's throw. I believe we can merge this as it is or after you've added the missin translation for the already confirmed speaker

Reply:
I don't like that tbh